docs/build/guides/conventions/workspace.mdx

```diff
@@ -0,0 +1,357 @@
+---
+title: Workspaces
+hide_table_of_contents: true
+description: Organize contracts using Cargo workspaces
+---
+
+### Initializing workspace project
+
+Using Cargo's workspace [feature](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html) makes it very convenient to organize your smart contracts in subdirectories of your project's root.
+
+It's very simple to get started using the cli:
+
+```
+stellar contract init . --name add_contract
+```
+
+Running this command will create a root project directory (`soroban-project`) and then initialize a project workspace with a single contract named `add_contract`.
+
+Adding one more contract template to the project can be done using the same command:
+
+```
+stellar contract init . --name main_contract
+```
+
+The project tree will look like this:
+
+```
+$ tree
```

Comment:
The tree command won't work for all users and could be a point of confusion. The main goal of displaying the file hierarchy is to communicate what it is and we can do that without the tree command. Other docs where we display the hierarchy we drop the command.
```suggestion
```